uui-components/src/pickers/PickerInputBase.tsx

```diff
@@ -203,6 +208,7 @@ export abstract class PickerInputBase<TItem, TId, TProps> extends PickerBase<TIt
             })),
             onKeyDown: e => this.handlePickerInputKeyboard(rows, e),
             fixedBodyPosition: this.props.fixedBodyPosition,
+            bodyCx: this.props.bodyCx,
```

Comment:
we need to have cx only on root node, not in nested DatePickerBody.

Reply:
done